properties/test_pandas_roundtrip.py

```diff
@@ -132,3 +132,12 @@ def test_roundtrip_pandas_dataframe_datetime(df) -> None:
     roundtripped.columns.name = "cols"  # why?
     pd.testing.assert_frame_equal(df, roundtripped)
     xr.testing.assert_identical(dataset, roundtripped.to_xarray())
+
+
+def test_roundtrip_1d_pandas_extension_array() -> None:
```

Comment:
does this cover the `Series.name` piece too?

Reply:
That's what caught it! So that codepath is in here.  I can add a test for that, although it will look basically identical (needs to be a 1D array to catch the series business).

Reply:
Let me know what you think there.  Could just add a comment here about that.  Or parametrize with more than an extension array as input.

Reply:
No as long as its checked 👍🏾